CIP-0018/CIP-0018.md

```diff
@@ -0,0 +1,104 @@
+
+Even for smaller actors, it can be interesting to delegate to multiple pools to limit risks. Pools may underperform or simply change their parameters from a day to another (for which wallets still to not warn users about). Delegating to more pools can reduce the impact of pool failure (for one or more epochs) or unattended changes in pool fees. It may as well be a matter of choice if users do want to delegate to several independent entities for various other reasons.
+
+Another concern regards privacy leaks coming with the existing wallet scheme. Since a same stake key is associated to every single address of the wallet, it creates a kind of watermark that allows for tracing all funds belonging to a same wallet very easily (it suffices to look at the stake part of addresses). By allowing the wallet to hold multiple stake keys, rotating through them when creating changes does make the traceability a bit harder. One could imagine using a hundred of stake keys delegated to the same pool.
+
+# Specification
+
+### Overview
+
+The restriction from [CIP-0011] regarding the derivation index reserved for stake key is rendered obsolete by this specification. That is, one is allowed to derive indexes beyond the first one (index = 0) to effectively administrate multi-stake keys accounts. The creation of new stake keys is however tightly coupled to the registration of associated stake keys to allow wallet software 
+to automatically discover stake keys on chain. In this design, stake key indexes form *at all time* a contiguous sequence with no gap.
+
+### Key registration
+
+The derivation of new stake keys should abide by the following rules:
+
+- **1.A)** Stake keys must be derived sequentially, from 0 and onwards.
+- **1.B)** Except for the first stake key (index = 0), stake keys should be registered before being used in output addresses.
+
+The registration of stake keys beyond the first must satisfy one of the following conditions:
+
+- **2.A)** It must exist an active<sup><a href="#1-active-stake-key">1</a></sup> stake key registration certificate on chain of the immediately previous stake key, located in the stable area<sup><a href="#2-stable-area">2</a></sup> of the chain. 
+- **2.B)** It must exist another stake key registration certificate of the immediately previous stake key in the same transaction which satisfies 2.A or 2.B.
+
+Said differently, one can only register a new stake key if previous keys have already been registered and are immutable. We also allow registering keys in batch to alleviate the user experience for users wanting to register many stake keys.
```

Comment:
Does it mean one will be able to make multi-delegation only at least every 36h?

Reply:
Yes and no. It means that you'll be to register new stake keys after at least 36h from the last registration. Yet, you can register many keys at once. So, if you have a portfolio of let's say 10 pools, you can multi-delegate in one go. But then, once done if you want to add a 11th pool, you'll have to wait 36h. However, you can change the delegation settings of any of these pools anytime you want. What's really constrained is the registration (and conversely, de-registration) of stake keys.

Reply:
ok, thanks. Good, that's how I understood it.